### PR TITLE
💄 style: add claude 3.5 haiku model

### PR DESCRIPTION
**Pull request by LovelyGuYiMeng**
#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [x] 💄 style
- [ ] 👷 build
- [ ] ⚡️ perf
- [ ] 📝 docs
- [ ] 🔨 chore

### CODE REVIEW

File: src/config/modelProviders/anthropic.ts [hunk 1]
```diff
@@ -3,6 +3,24 @@ import { ModelProviderCard } from '@/types/llm';
 // ref: https://docs.anthropic.com/en/docs/about-claude/models#model-names
 const Anthropic: ModelProviderCard = {
   chatModels: [
+    {
+      description:
+        'Claude 3.5 Haiku 在许多评估中与我们的前一个最大模型 Claude 3 Opus 的性能相当，速度与上一代 Haiku 类似。',
```
Comment (arvinxx) [inline src/config/modelProviders/anthropic.ts]: 'Claude 3.5 Haiku 是 Anthropic 最快的下一代模型。与 Claude 3 Haiku 相比，Claude 3.5 Haiku 在各项技能上都有所提升，并在许多智力基准测试中超越了上一代最大的模型 Claude 3 Opus。'

**Review by arvinxx — COMMENTED** [review]
另外是否 haiku-3 不默认显示？

File: src/config/modelProviders/anthropic.ts [hunk 2]
```diff
@@ -3,6 +3,24 @@ import { ModelProviderCard } from '@/types/llm';
...
+      functionCall: true,
+      id: 'claude-3-5-haiku-20241022',
+      maxOutput: 8192,
+      pricing: {
+        cachedInput: 0.1,
+        input: 1,
+        output: 5,
+        writeCacheInput: 1.25,
+      },
+      releasedAt: '2024-10-22',
```
Comment (arvinxx) [inline src/config/modelProviders/anthropic.ts]: 其实应该是 11-05 release 的…

Comment (LovelyGuYiMeng) [reply src/config/modelProviders/anthropic.ts]: 🫠

**Comment by SpeedupMaster** [comment]
Claude 3.5 Haiku is available across our first-party API, Amazon Bedrock, and Google Cloud’s Vertex AI—**initially as a text-only model** and with image input to follow.
暂时还不支持图片输入

**Pull request merged**